src/test/java/io/confluent/kafkarest/unit/ConsumerManagerTest.java

```diff
@@ -172,38 +177,50 @@ public void testConsumerNormalOps() throws InterruptedException, ExecutionExcept
     String cid = consumerManager.createConsumer(
         groupName, new ConsumerInstanceConfig(EmbeddedFormat.BINARY));
     sawCallback = false;
+    actualException = null;
```

Comment:
@ewencp What would be reported if a test failed?  If its enough to identify the assert that failed then it is worth a go but if you get what amounts to an unexpected exception in the test I'd probably advise against it.  At least with the current approach you can setup asserts clearly to identify what went wrong.